node-fs-readfile/cat.ts

```diff
@@ -0,0 +1,18 @@
+import { readFile } from 'node:fs/promises';
+
+try {
+  const filenames: string[] = process.argv.slice(2);
```

Comment:
Just a reminder that it is ok for the system to use type inference to know what type is being assigned to a variable. TypeScript inference will just alert us later if we try to assign a different type later.